runtime-transaction/benches/process_compute_budget_instructions.rs

```diff
@@ -158,12 +158,45 @@ fn bench_process_compute_budget_instructions_mixed(c: &mut Criterion) {
         );
 }
 
+fn bench_process_compute_budget_and_transfer_only(c: &mut Criterion) {
+    let num_instructions = 355;
+    let pubkey = Pubkey::new_unique();
+    c.benchmark_group("bench_process_compute_budget_and_transfer_only")
+        .throughput(Throughput::Elements(NUM_TRANSACTIONS_PER_ITER as u64))
+        .bench_function(
+            format!("{num_instructions} transfer instructions and compute budget ixs"),
+            |bencher| {
+                let payer_keypair = Keypair::new();
+                let mut ixs: Vec<_> = (4..num_instructions)
+                    .map(|i| system_instruction::transfer(&payer_keypair.pubkey(), &pubkey, i))
+                    .collect();
+                ixs.extend(vec![
+                    ComputeBudgetInstruction::request_heap_frame(40 * 1024),
+                    ComputeBudgetInstruction::set_compute_unit_limit(u32::MAX),
+                    ComputeBudgetInstruction::set_compute_unit_price(u64::MAX),
+                    ComputeBudgetInstruction::set_loaded_accounts_data_size_limit(u32::MAX),
+                ]);
```

Comment:
won't this be 359 ixs?

Reply:
it starts from `4` 😈  I shall change it to `let mut ixs: Vec<_> = (0..num_instructions - 4)` for readability

Reply:
=o